common/lib/dependabot/errors.rb

```diff
@@ -661,6 +661,23 @@ def initialize(dependencies)
     end
   end
 
+  class UnresolvableVersionError < DependabotError
+    extend T::Sig
+
+    sig { returns(T::Array[String]) }
+    attr_reader :dependencies
+
```

Comment:
Thanks for adding a new class; I think you'll have to make some changes on the API end too to get this error to sho up. @kbukum1 , can you share some sample past PRs?